### PR TITLE
Fix/9680 Update UI to match the Design

### PR DESCRIPTION
**Pull request by 30mar**
## Description
Update main text to match figma design

## Link to Jira
https://jira-ibs.wbs.net.sap/browse/EXPOSUREAPP-9680

## Screenshots
Simulator Screen Shot - iPhone 11 - 2022-06-27 at 03 39 20

### CODE REVIEW

**Review by naveeddotio — APPROVED** [review]

File: src/xcode/ENA/ENA/Source/Scenes/HealthCertificates/HealthCertificate/HealthCertificateViewModel.swift [hunk 1]
```diff
@@ -143,14 +143,31 @@ final class HealthCertificateViewModel {
 			title = AppStrings.HealthCertificate.Details.euCovidCertificate
 			subtitle = AppStrings.HealthCertificate.Details.RecoveryCertificate.title
 		}
+		let attributedTitle = NSAttributedString(
+			string: title,
+			attributes: [
+				.font: UIFont.enaFont(for: .body),
+				.foregroundColor: UIColor.enaColor(for: .textContrast),
```
Comment (flxschmidt) [inline src/xcode/ENA/ENA/Source/Scenes/HealthCertificates/HealthCertificate/HealthCertificateViewModel.swift:150]: nit pik:

`UIColor.` implicite

**Review by flxschmidt — APPROVED** [review]
lgtm, some optional comments

File: src/xcode/ENA/ENA/Source/Scenes/HealthCertificates/HealthCertificate/HealthCertificateViewModel.swift (hunk 1 above)
Comment (30mar) [reply src/xcode/ENA/ENA/Source/Scenes/HealthCertificates/HealthCertificate/HealthCertificateViewModel.swift:150]: it wouldn't compile, i get a compiler error "Reference to member 'body' cannot be resolved without a contextual type"

Comment (30mar) [reply src/xcode/ENA/ENA/Source/Scenes/HealthCertificates/HealthCertificate/HealthCertificateViewModel.swift:150]: it wouldn't compile, i get a compiler error "Reference to member 'body' cannot be resolved without a contextual type" if i use  .body or .textContrast

Comment (naveeddotio) [reply src/xcode/ENA/ENA/Source/Scenes/HealthCertificates/HealthCertificate/HealthCertificateViewModel.swift:150]: @30mar you will write .enaColor(for: .textContrast) or .enaFont(for: .body), it should work.

**Review by nickguendling — APPROVED** [review]

**Pull request merged**